[2022-12-13T16:39:40.144+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: bus_pos_etl.etl_bus_loc manual__2022-12-13T16:39:36.499207+00:00 [queued]>
[2022-12-13T16:39:40.161+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: bus_pos_etl.etl_bus_loc manual__2022-12-13T16:39:36.499207+00:00 [queued]>
[2022-12-13T16:39:40.162+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-13T16:39:40.162+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-13T16:39:40.162+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-13T16:39:40.184+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): etl_bus_loc> on 2022-12-13 16:39:36.499207+00:00
[2022-12-13T16:39:40.195+0000] {standard_task_runner.py:55} INFO - Started process 1459 to run task
[2022-12-13T16:39:40.200+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bus_pos_etl', 'etl_bus_loc', 'manual__2022-12-13T16:39:36.499207+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/test_noe_bash.py', '--cfg-path', '/tmp/tmp_nh2oheb']
[2022-12-13T16:39:40.203+0000] {standard_task_runner.py:83} INFO - Job 22: Subtask etl_bus_loc
[2022-12-13T16:39:40.280+0000] {task_command.py:389} INFO - Running <TaskInstance: bus_pos_etl.etl_bus_loc manual__2022-12-13T16:39:36.499207+00:00 [running]> on host ff6f332d28b5
[2022-12-13T16:39:40.391+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=noe
AIRFLOW_CTX_DAG_ID=bus_pos_etl
AIRFLOW_CTX_TASK_ID=etl_bus_loc
AIRFLOW_CTX_EXECUTION_DATE=2022-12-13T16:39:36.499207+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-13T16:39:36.499207+00:00
[2022-12-13T16:39:40.394+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-13T16:39:40.396+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/etl/etl.py']
[2022-12-13T16:39:40.421+0000] {subprocess.py:86} INFO - Output:
[2022-12-13T16:39:43.277+0000] {subprocess.py:93} INFO - bus_pos.csv création du blob en cours
[2022-12-13T16:39:43.278+0000] {subprocess.py:93} INFO - blob créé
[2022-12-13T16:39:43.278+0000] {subprocess.py:93} INFO - upload du csv vers le blob en cours
[2022-12-13T16:39:43.458+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-13T16:39:43.488+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=bus_pos_etl, task_id=etl_bus_loc, execution_date=20221213T163936, start_date=20221213T163940, end_date=20221213T163943
[2022-12-13T16:39:43.515+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-13T16:39:43.538+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
